smart-contracts/contracts-common/concordium-contracts-common/src/types.rs

```diff
@@ -653,7 +653,15 @@ pub struct CredentialSignatures {
 #[cfg_attr(feature = "fuzz", derive(Arbitrary))]
 pub struct Timestamp {
     /// Milliseconds since unix epoch.
-    pub(crate) milliseconds: u64,
```

Comment:
I can't find the FromStr implementation for this and the base timestamp has it.

I think ti should support parsing both from a u64 as milliseconds and the RFC... format (Dual to display).

Reply:
The implementation can be found here https://github.com/Concordium/concordium-base/pull/499/files#diff-5b114d32401abf835eb7e1165d42c0d36970eab7c96eda1d0e121601fe7e105dR794
And it already supports both u64 strings and the RFC... format.